public/js/main.js

```diff
@@ -288,10 +289,17 @@ function setFileURLs(new_provider) {
       appLoading.stop();
     } else if (lastQuery !== val) {
       animateTop();
-      index.search(val, displayMatchingLibraries);
+      displayItems = 20;
+      index.search(val, {hitsPerPage: displayItems}, displayMatchingLibraries);
     }
     lastQuery = val;
   }
+  $(window).scroll(function() {
+    if($(window).scrollTop() + $(window).height() === $(document).height()) {
```

Comment:
What if `if ($(window).scrollTop() + $(window).height() * 2 >= $(document).height())`?
Trying to load more result when `$(window).height()` px away from bottom.